src/csvcubed/utils/sparql_handler/csvw_state.py

```diff
@@ -57,3 +70,12 @@ def get_primary_catalog_metadata(self) -> CatalogMetadataResult:
         raise KeyError(
             f"Could not find catalog metadata in primary graph '{self.primary_graph_uri}'."
         )
+
+    def get_column_definitions_for_csv(self, csv_url: str) -> List[ColumnDefinition]:
+        """
+        Returns the `ColumnDefinition`s for a given csv file.
```

Comment:
Can you mention that it raises a key error if it can't be found (inside the docstring comment)?